src/test/java/pages/DashboardPage.java

```diff
@@ -0,0 +1,196 @@
+	
+	@FindBy(xpath="//button/span[contains(text(), 'Ok')]")
+	public WebElement ClickOkonLeaveGamePopUp;
+	
+	@FindBy(className="profile-pic")
+	public WebElement UserProfilePic;
+	
+	@FindBy(xpath="//*[contains(text(),'Fastest')]")
+	public WebElement FastestFingerGameButton;
+	
+	@FindBy(xpath="(//*[contains(text(),'Flag as inappropriate')])[1]")
+	public WebElement FlagAsInappropriateButton;
+	
+	@FindBy(xpath="//*[contains(text(),'Question is not relevant')]")
+	public WebElement SelectFlagQuestionOption;
+	
+	@FindBy(xpath="//*[contains(text(),'Submit')]")
+	public WebElement ClickSubmitButton;
+
+	@FindBy(xpath="(//*[contains(text(),'Upvote')])[1]")
+	public WebElement UpvoteQuestion;
+	
+	@FindBy(xpath="//*[contains(@class,'games-count num-container')]")
+	public WebElement NoOfGames;
+	
+	@FindBy(xpath="//div[@class='row question-count']/div/div/h4")
+	public WebElement QuestionNumber;
+	
+	public DashboardPage(WebDriver driver) {
+		this.driver=driver;
+		PageFactory.initElements(this.driver, this);
+	}
+	
+	public void DashboardButton() {
+		dashboardbutton.click();
+		}
+	
+// Click on Play button of topic 	
+	public void PlayOnFirstTopic() {
+		landsOnGamesScreen.click();
+		}
+// Click on Single Player Button
+	public void ClickSinglePlayerButton() {
+		SinglePlayerButton.click();
+	}
+//Select first option	
+	public void Selectoption() {
+		Option.click();
+	}
+//Click on Random Opponent Button 
+	public void initiateRandomGame() {
+		RandomGameButton.click();
+	}
+//Click on Rapid Fire Game Button	
+	public void initiateFastestFinger() {
+		FastestFingerGameButton.click();
+	}
+//Click on Fastest Finger game button 
+	public void initiateRapidFire() {
+		RapidfireGameButton.click();
+	}
+	
+//Click on Flag as inappropriate button
+	public void flagquestion() {
+		FlagAsInappropriateButton.click();
+		SelectFlagQuestionOption.click();
+		ClickSubmitButton.click();
+		}
+//Upvote a Question from result screen 
+	public void upvotequestion() {
+		UpvoteQuestion.click();		
+	}
+//Leave game
+	public void leavegame() {
+		LeaveGameButton.click();
+	}
+//Wait for Looking for Opponent to disappear
+	public void waittostartgame() {
+		new WebDriverWait(driver,40).until(ExpectedConditions.invisibilityOfElementLocated(By.xpath("(//*[contains(text(),'Looking')])")));
+	}
+	
+//Click Ok on Leave Game Pop Up
+	public void popup() {
+		String myWindowHandle = driver.getWindowHandle();
+		driver.switchTo().window(myWindowHandle);
+		ClickOkonLeaveGamePopUp.click();
+	}
+	
+//Check that user is landed successfully on Result Screen 
+	public void userprofilepicture() {
+		UserProfilePic.click();
+	}
+//Get Number of games Played by user	
+	public void getgamesplayed() {
+		String gamesplayed = NoOfGames.getText();
+        System.out.println(gamesplayed);   	
+	}
+	
+//Play Game for FF	
+	 public void playgameFF()
+	 {   	 
+		 int ques=0;
+		 String quesno = QuestionNumber.getText();
+		 System.out.println(quesno);
```

Comment:
Comment out all the print statements from page classes

Reply:
Done